python/mlc_chat/cli/chat.py

```diff
@@ -0,0 +1,47 @@
+"""Command line entrypoint of chat."""
+from mlc_chat.help import HELP
+from mlc_chat.interface.chat import ChatConfigOverride, chat
+from mlc_chat.support.argparse import ArgumentParser
+
+
+def main(argv):
+    """Parse command line arguments and call `mlc_llm.interface.chat`."""
+    parser = ArgumentParser("MLC LLM Chat CLI")
+
+    parser.add_argument(
+        "model",
+        type=str,
+        help=HELP["model"] + " (required)",
+    )
+    parser.add_argument(
+        "--opt",
+        type=str,
+        default="O2",
+        help=HELP["opt"] + ' (default: "%(default)s")',
+    )
+    parser.add_argument(
+        "--device",
+        type=str,
+        default="auto",
+        help=HELP["device_deploy"] + ' (default: "%(default)s")',
+    )
+    parser.add_argument(
```

Comment:
Hey I'd love to keep things consistent, so would you like to see [this comment](https://github.com/mlc-ai/mlc-llm/pull/1564#discussion_r1446413407)?

Reply:
Do you think we should compress all stuff in `--overrides` or expand to `---conv-template`, `--tensor-parallel-shards`, etc?